packages/node/README.md

```diff
@@ -20,6 +20,12 @@ Some specific examples of this include:
 
 We have [some diagrams](./docs/diagram.md) explaining the Node's architecture and control flow.
 
+## Node Address and Signing Keys
+
+The Node is expected to have access to an [extended key](https://bitcoin.org/en/glossary/extended-key) which it uses to derive a "public identifier" used as the address of a Node instance. This key is also used to produce private keys at app-specific derivation paths.
```

Comment:
access to an extended public key (even the linked page says "extended key" is ambiguous)

Reply:
Updated README